snuba/datasets/errors_replacer.py

```diff
@@ -281,13 +282,23 @@ def get_projects_query_flags(
                 exclude_groups_key, 0, 0, withscores=True,
             )
 
-    with sentry_sdk.start_span(op="function", description="execute_redis_pipeline"):
+    with sentry_sdk.start_span(
+        op="function", description="execute_redis_pipeline"
+    ) as span:
         results = p.execute()
+        span.set_tag("results", results)
+        span.set_tag("results_size", sys.getsizeof(str(results)))
```

Comment:
Got the size of `str(results))` since `sys.getsizeof()` does not count recursively.

Reply:
Put that in a comment